### PR TITLE
[SC-166717] Support 3.10 and 3.11

### PR DESCRIPTION
**Pull request by qcastel**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Version 3.0.0 - Feature release
+
+ - Add more supported Python versions. This plugin can now 3.10 and 3.11
```
Comment (thtrunck) [inline Changelog.md:5]: You are missing a word

Comment (qcastel) [reply Changelog.md:5]: note: this PR is pending work still, waiting to see if we can drop 2.7 or not before investing more time on it

**Review by thtrunck — APPROVED** [review]
Except the missing word, I fine with the change. But we don't have release branch in this repo so we should wait until 12.6 is release.

**Comment by amandineslx** [comment]
just ran into this PR that we can merge now 12.6 is released

**Pull request merged**